### PR TITLE
Made description complusory

### PR DESCRIPTION
**Pull request by AbhishekDeb70**

### CODE REVIEW

File: client/src/components/Admin/Products/AddForm.js [hunk 1]
```diff
@@ -35,15 +35,19 @@ const AddForm = () => {
     if (!Object.keys(details).includes("name") || details.name === "") {
       setErrormsg("Name cannot be empty");
       return;
-    } else if (categoryid === "") {
+    }else if (fileData.length == 0){
```
Comment (iMagesh) [inline client/src/components/Admin/Products/AddForm.js:38]: align it neatly

File: client/src/components/Admin/Products/AddForm.js [hunk 2]
```diff
@@ -35,15 +35,19 @@ const AddForm = () => {
...
+     else if (categoryid === "") {
       setErrormsg("Category cannot be empty");
       return;
     } else if (!Object.keys(details).includes("price") || details.price === "") 
     {
       setErrormsg("Price cannot be empty");
       return;
-    }else if (fileData.length == 0){
-      setErrormsg("Please select an image");
+    } else if (!Object.keys(details).includes("description") || details.description === ""){
```
Comment (iMagesh) [inline client/src/components/Admin/Products/AddForm.js:49]: here again alignment is not good, fix it

**Comment by iMagesh** [comment]
@AbhishekDeb70 resolve the comments quickly let's merge now

**Comment by AbhishekDeb70** [comment]
> @AbhishekDeb70 resolve the comments quickly let's merge now

Preetified it

**Comment by iMagesh** [comment]
@AbhishekDeb70 setup prettifier on vscode to always prettify files on save

**Pull request merged**